packages/adapter-edgedb/dbschema/default.esdl

```diff
@@ -0,0 +1,71 @@
+module default {
```

Comment:
We don't need this file in the adapter package, right?
Everything under `/dbschema` can be removed IMO